torchani/__init__.py

```diff
@@ -42,8 +43,8 @@
            'utils', 'neurochem', 'models', 'optim']
 
 try:
-    from . import ase  # noqa: F401
-    __all__.append('ase')
+    from . import ase_ani  # noqa: F401
```

Comment:
Can you do `from . import ase_ani as ase` then to avoid API change?

Reply:
Yes! I can do that.

Reply:
I reverted the change, it seems like the issue only happens if you try to run a script directly and the script is in the same directory as ase.py, it doesn't seem to affect the program at this point so I will leave it like it is. I had the issue while I was working on this because I was constantly running 'units.py' as a script, but since it doesn't affect us now its not worth it to change the name.